CMSgov/design-system
# [NO-TICKET] DSM content migration for mgov consistent header

## Summary

- Additional content and imagery migrated from the DSM
- This content was missed previously (it was in a tab and not main content on the page) thanks @brentschneider 

## How to test

1. build and run the doc site `yarn build && yarn start` and visit the mgov header page to see the additional content 

## Checklist




### If this is a change to code:


### If this is a change to documentation:
